justfile

```diff
@@ -0,0 +1,64 @@
+set dotenv-load
+
+# package name
+PACKAGE_NAME := env("PACKAGE_NAME", "tedge-mosquitto")
```

Comment:
`env` is not working with just 1.13.0.

`env_var_or_default` is deprecated but works fine: this is the only newish command used by this justfile.

Reply:
Though I think its ok to focus on newer versions for build tooling, otherwise it will break at some point when the deprecated functions are removed.